modules/serverless-rn-1-27-0.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-27_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.27
+
+{ServerlessProductName} 1.27 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-27_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.6.
+* {ServerlessProductName} now uses Knative Eventing 1.6.
+* {ServerlessProductName} now uses Kourier 1.6.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.6.
+* {ServerlessProductName} now uses Knative Kafka 1.6.
+* The `kn func` CLI plug-in now uses `func` 1.8.1.
+
+* {ServerlessProductName} 1.27 is the earliest release that is fully supported on {product-title} 4.12. {ServerlessProductName} 1.25 and older does not deploy on {product-title} 4.12.
++
+Consequently, before upgrading {product-title} to version 4.12, first upgrade {ServerlessProductName} to version 1.26 or newer.
+
+* Namespace-scoped brokers are now TP. Such brokers can be used, for instance, to implement role-based access (RBAC) policies.
+
+* `KafkaSink` now uses the `CloudEvent` binary content mode by default. The binary content mode is more efficient than the structured mode because it uses headers in its body instead of a `CloudEvent`. For example, for the HTTP protocol, it uses HTTP headers.
+
+* You can now use the gRPC framework over the HTTP/2 protocol for external traffic using the OpenShift Route on {product-title} 4.10 and later. This improves efficiency and speed of the communications between the client and server.
+
+* API version `v1alpha1` of the Knative Operator Serving and Eventings CRDs is deprecated in 1.27. It will be removed in future versions. Users are strongly recommended to use the `v1beta1` version instead. This does not affect the existing installations, because CRDs are updated automatically when upgrading the Serverless Operator.
+
+* The delivery timeout feature is now enabled by default. It allows you to specify the timeout for each sent HTTP request. The feature remains a TP.
+
+* The `serverless.openshift.io/enable-secret-informer-filtering` annotation for the Serving CR is now deprecated and will be removed in {ServerlessProductName} 1.28. The secret filtering configuration will be done using an environment variable.
```

Comment:
the enable-secret-informer-filtering annotation deprecation should also appear in the "Table 2. Deprecated and removed features tracker "

Reply:
I think the important point of https://issues.redhat.com/browse/SRVKS-944 is that the filtering will be enabled by default in 1.28. To prepare for 1.28, the user using servicemesh integration should start labeling the secrets used by domainmapping as documented in  https://docs.openshift.com/container-platform/4.12/serverless/security/serverless-custom-domains.html#serverless-domain-mapping-custom-tls-cert_serverless-custom-domains

Reply:
Yes, actually deprecation of `serverless.openshift.io/enable-secret-informer-filtering` does not have an impact because currently users use it only for enabling.

We should document https://issues.redhat.com/browse/SRVKS-947 before this release note. Then, we need to annonce:
* We will enable secret informer by default so users need to add the label (`certificate-uid`) to DomainMapping.
* If you cannot add the label, you can disable the secret informer by following the section which will be added in SRVKS-947.

Reply:
We could remove the notes for 1.27 completely. Announce deprecation for 1.28:
>  The `serverless.openshift.io/enable-secret-informer-filtering` annotation for the Serving CR is now deprecated and it will be removed in future versions. The secret filtering configuration should be done using an environment variable (add link to the new way).

Document the new way as described in SRVKS-947 (1.28).

I suspect this is what @nak3 described.

Reply:
We don't want to break users that don't currently use secret filtering when they automatically upgrade to 1.28. So we have to tell them now to either start labeling the secrets used in domainmapping with 1.27, or add the workload override with 1.27 to disable the filtering, so then an automatic 1.28 upgrade won't break them.

Reply:
I agree with Stavros, would be good to mention deprecation together with documenting the new way (1.28). users than can label the secret or use workload overrides with 1.28 and get ready for future releases when the old way will be removed.

Reply:
@mgencur @maschmid Would it be too early to mention that we enable the feature by default in 1.28? See the question [here](https://github.com/openshift-knative/serverless-operator/pull/1954#issuecomment-1418730109).

Reply:
Resolved by removing the note entirely per Stavros' suggestion.